src/components/TeamMember.astro

```diff
@@ -27,10 +27,10 @@ const { member, index, kind } = Astro.props;
       width={200}
     />
   </a>
-  <h4
```

Comment:
same as above

Reply:
Same [reason as above](https://github.com/biznaga-fest/web/pull/8#discussion_r1769565267), but here there was also a heading jump from h2 to h4:

Reply:
Its not a paragraph, though. Use h3 instead

Reply:
Done at [b0df69b0ec652dd5f866913fc00ffc4144cddebc ](https://github.com/biznaga-fest/web/pull/8/commits/b0df69b0ec652dd5f866913fc00ffc4144cddebc)